Assets/StreamingAssets/Data/Headlines.xml

```diff
@@ -4,4 +4,18 @@
     <Headline>Notice: Quillcorp has placed an embargo on "Chairs", any Quillcorp Basic Utility Station found in possession of this illegal contraband will be fined 2 million Quillbucks.</Headline>
     <Headline>Introducing the MagBlast 2200 magnetic rifle. 70% of the time, it hits every time.</Headline>
     <Headline>Thinktank finds something that could go wrong. Head of QuillCorp not available for comment.</Headline>
+    <Headline>Attention: Quillcorp has banned all video games except for ProjectPorcupine. Any person caught playing a banned game will face severe penalties.</Headline>
+    <Headline>As the inevitability of his impending death began to crystallize in his mind, Rupert unaccountably wondered whether it would be classified as "W40.9XXD Explosion of unspecified explosive materials (subsequent encounter)" or, he hoped, the more colourful "W59.22XA Struck by turtle (initial encounter)" under the revised and expanded ICD-10-CM American medical billing code scheme.</Headline>
```

Comment:
"As the inevitability.." isn't really a headline, I think its a bit too long and doesn't fit the rest of the additions